content/en/infrastructure/containers/container_images.md

```diff
@@ -0,0 +1,171 @@
+
+**Note**: CSM Vulnerabilities is not available for AWS Fargate or Windows environments.
+
+{{< tabs >}}
+{{% tab "Kubernetes (Helm)" %}}
+
+If you are using Helm version `>= 3.46.0`, image collection is [enabled by default][1].</br>
+Or, add the following to your `values.yaml` Helm configuration file:
+
+```yaml
+datadog:
+  containerImageCollection:
+    enabled: true
+  sbom:
+    containerImage:
+      enabled: true
+```
+[1]: https://github.com/DataDog/helm-charts/blob/main/charts/datadog/values.yaml#L651
+{{% /tab %}}
+
+{{% tab "Kubernetes (Operator)" %}}
+
+Add the following to the spec section of your `values.yaml` file:
+
+```yaml
+apiVersion: datadoghq.com/v2alpha1
+kind: DatadogAgent
+metadata:
+  name: datadog
+spec:
+  features:
+    # ...
+    sbom:
+      enabled: true
+      containerImage:
+        enabled: true
+```
+
+{{% /tab %}}
+
+{{% tab "ECS EC2" %}}
+
+To enable container image vulnerability scanning on your [ECS EC2 instances][1], add the following environment variables to your `datadog-agent` container definition:
+
+```yaml
+{
+    "containerDefinitions": [
+        {
+            "name": "datadog-agent",
+             ...
+            "environment": [
+              ...
+              {
+                "name": "DD_CONTAINER_IMAGE_ENABLED",
+                "value": "true"
+              },
+              {
+                "name": "DD_SBOM_ENABLED",
+                "value": "true"
+              },
+              {
+                "name": "DD_SBOM_CONTAINER_IMAGE_ENABLED",
+                "value": "true"
+              }
+            ]
+        }
+    ]
+  ...
+}
+```
+
+If the Agent fails to extract the SBOM from the container image, increase the Agent memory in the container definition:
+
+```yaml
+{
+    "containerDefinitions": [
+        {
+            "name": "datadog-agent",
+            "memory": 256,
+            ...
+        }
+     ]
+    ...
+}
+```
+[1]: https://docs.datadoghq.com/containers/amazon_ecs/?tab=awscli#setup
+
+{{% /tab %}}
+
+{{% tab "Hosts" %}}
+
+Add the following to your `datadog.yaml` configuration file:
+
+```yaml
+sbom:
+  enabled: true
+  container_image:
+    enabled: true
+container_image:
+  enabled: true
+```
+
+[1]: /containers/amazon_ecs/?tab=awscli#setup
+{{% /tab %}}
+{{< /tabs >}}
+
+### Container registries
+
+#### Amazon Elastic Container Registry (Amazon ECR)
+
+Set up the [AWS integration][4] to begin crawling Container Image metadata from Amazon ECR.
+
+## Container image tagging
+
+Containers are [tagged][6] with all existing host-level tags, as well as with metadata associated with individual containers. All containers are tagged by `image_name`, including integrations with popular orchestrators, such as ECS and Kubernetes, which provide further container-level tags. 
+If you have a configuration for [Unified Service Tagging][7] in place, Datadog automatically picks up `env`, `service`, and `version` tags. 
```

Comment:
Container Image tagging is not related to Live Container tags. As of today, if users want to tag their Images with arbitrary tags, the only supported way is by using the "Extract Labels as Tags" on the agent. These tags will then be picked by the Container Image check.
https://docs.datadoghq.com/containers/docker/tag/?tab=containerizedagent#extract-labels-as-tags

Reply:
Would it be possible to add a `team` tag on the screenshot you've shared?

Eventually, we'd like to add a link to this doc from the Container Images view when the Team Lens is enabled so having an example with team will be useful

![Container Images Datadog 2023-12-20 at 3 41 37 PM](https://github.com/DataDog/documentation/assets/45740044/c7efce9c-40a4-4d1d-aed6-cd745f210722)

Reply:
Hi @moisesbotarro for the screenshot, are you wanting just a screen shot of the Container Images view with a team filter (such as below)? Or were you wanting a screenshot of the details side panel of a specific container where it has an arbitrary `team` container tag? I'm not finding an example of the latter, i.e. if I filter by `team:mobile` and then select one of those images, the `team` tag doesn't seem to carry over. Perhaps because "extract labels as tags" isn't configured. Let me know if this is what you were looking for if you have an example of one ? Thanks!

<img width="1839" alt="Container Images Datadog 2023-12-20 at 1 35 49 PM" src="https://github.com/DataDog/documentation/assets/26307719/f2f2931d-3018-4dfe-82ac-2697991be0f9">

Reply:
HI @aliciascott, what I had in mind is to show arbitrary tags in the tag list on the details side panel like the screenshot you had before.


> I'm not finding an example of the latter, i.e. if I filter by team:mobile and then select one of those images, the team tag doesn't seem to carry over.

This is expected because this `team` tag is coming from the containers running the image and it's not a tag of the image itself.

> Perhaps because "extract labels as tags" isn't configured.

You are right. I was looking for other examples and I don't think that "extract labels as tags" is set up yet.

What do you think of only mentioning that users can enrich their container images with arbitrary tags and link to the "Extract Labels as Tags" doc, without any screenshot?

Reply:
Sounds good to me, per our slack convo, we can work on setting this up later and iterate on the docs.